main.go

```diff
@@ -228,6 +233,21 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err = (&controllers.HelmRepositoryOCIReconciler{
+		Client:                  mgr.GetClient(),
+		EventRecorder:           eventRecorder,
+		Metrics:                 metricsH,
+		Getters:                 getters,
+		ControllerName:          controllerName,
+		RegistryClientGenerator: util.RegistryClientGenerator,
+	}).SetupWithManagerAndOptions(mgr, controllers.HelmRepositoryReconcilerOptions{
+		MaxConcurrentReconciles: concurrent,
+		RateLimiter:             helper.GetRateLimiter(rateLimiterOptions),
+	}); err != nil {
+		setupLog.Error(err, "unable to create controller", "controller", sourcev1.HelmRepositoryKind)
```

Comment:
Since there are two reconcilers watching the same Kind now, this log doesn't provide enough information about to uniquely identify which reconciler setup failed. Maybe we can add another field to uniquely identify the type of helm repository.